napari/layers/points/points.py

```diff
@@ -131,6 +131,12 @@ class Points(Layer):
     cache : bool
         Whether slices of out-of-core datasets should be cached upon retrieval.
         Currently, this only applies to dask arrays.
+    fixed_size : bool
+        If active, point sizes do not change when zooming.
+    antialias : float
+        If non-zero, defines the width in pixels of antialiasing.
```

Comment:
canvas pixels or pixels in world coordinates?

Reply:
similar to @sofroniewn - is there a sane default for this which prevents the need for exposing it? Under what situation would I be expected/likely to want to modify this myself?

Reply:
The best sane defaul that I can think of is to turn it on in 2D and off in 3D. Due to transparency being broken in vispy, this is especially obvious in 3D where markers change overlap order and everything looks weird. In 2D it's probably ok to leave it on, though if many overlapping points are present it still looks off (but less distracting, since they don't move relative to each other... unless `fixed_size` is on :P)

Reply:
hmm, ok that does sound a little complicated - but also potentially too complicated to be exposed in the main public API. What does @kevinyamauchi think about this? I'm still hoping we can maybe reduce all this down to one or two public modes

Reply:
I'm totally on board with hiding `antialias` behind a private property or something like that, but I think it should be accessible from napari (unlike now), at least until transparency rendering is fixed. The automatic switch is a decent compromise to avoid the exposure, but there are also many times when I used normal discs for points in 3D and still it looks bad.

Unless of course we decide to default to `spherical` being enabled in 3D (about which I'm not sure how I feel).